contrib/for-tests/volumes/gluster/run_gluster

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Any reason not to add a ".sh" to this file's name, to make it clearer from the outside what it is?